examples/gameroom/cli/src/main.rs

```diff
@@ -62,7 +65,12 @@ fn run() -> Result<(), CliError> {
 
     setup_logging(log_level);
 
-    Ok(())
+    match matches.subcommand() {
+        (action::database::SUBCMD, Some(matches)) => {
```

Comment:
Inline these strings, as it makes it more opaque, rather than clarifying it

Reply:
Done